gems/smithy/spec/fixtures/multi_protocol/model.smithy

```diff
@@ -0,0 +1,14 @@
+$version: "2"
+
+namespace smithy.ruby.tests
+
+use smithy.protocols#rpcv2Cbor
+
+// Define a fake protocol trait for use.
+@trait
+@protocolDefinition
+structure testProtocol {}
+
+@testProtocol
+@rpcv2Cbor
```

Comment:
May be easier if you just define two test protocols here - then you don't have to worry about cbor existing for this test, which seems to just be a resolution test?

Reply:
Yeah true, I could use another dummy protocol in place but I do like being able to check that protocols welds are working correctly here.

Reply:
Isn't that already checked when you run protocol tests and the plugin added?

Reply:
Ahhhh yeah, once protocol tests are done - that's true. I could modify this now to using fake protocol (and remove smithy-build.json) with anticipation of cbor protocol tests being done soon (but keep around to test weld ordering later)

Reply:
I dont think the protocol tests will cover default resolution?  They only (currently) ever cover single protocol cases.  And even with multiple protocols defined on a service, they would never test the default resolution - we would instead need to configure the correct protocol for each test on the client.

Reply:
I'm not suggesting that. This test can simply test resolution with two dummy protocols. Juli wants to test rpc v2 with welds. That is implicitly done through protocol test because they would not succeed otherwise.

Reply:
I was thinking about putting in a dummy protocol to replace the first one but then I have to create a dummy weld as well. For now - I'm going to leave it as it is until we implement/consider weld ordering.

Some test cases we want to cover are:
* Case: There are two protocols set on the service model but the merged protocol welds only has one protocol
  * Expected: Generated protocol plugin should be using what protocol weld is bringing
  
* Case: There are two protocol sets on the service model and the weld protocols supports them as well
  * Expected: Generated protocol plugin should determine the default protocol by the protocol order priority

* Case: No protocol weld, but service model has protocols defined
  * Expected: default is nil